### PR TITLE
Task/show warn false

### PR DESCRIPTION
**Pull request by alvarolmedo**

### CODE REVIEW

**Comment by nsano-rururu** [comment]
@alvarolmedo

The current elastalert2 has "elasticsearch>=7.0.0,<8.0.0", so it seems that you don't need to modify requirements.txt and setup.py.

File: requirements.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ boto3>=1.4.4
 cffi>=1.11.5
 configparser>=3.5.0
 croniter>=0.3.16
-elasticsearch>=7.0.0
+elasticsearch>=7.5.0
 envparse>=0.2.0
```
Comment (jertel) [inline requirements.txt:9]: If you can revert the es lib upgrade (see #90 for why this is needed) we can get this merged in. If there is any applicable doc updates that might be helpful as well.

**Review by jertel — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (jertel) [reply requirements.txt]: Nevermind, I was able to revert it.

**Comment by alvarolmedo** [comment]
Sorry to be late, thank you. :)

**Pull request merged**